README.md

```diff
@@ -239,12 +238,25 @@ bower install edx-pattern-library --save
 
 - - -
 
-### Step 3: Reference UX Pattern Library Partials in your Sass Compile
+### Step 2: Use the UX Pattern Library in your project's Styling
 
-You can now use edX UX Pattern Library Sass/styling in your project. To do so,
-just import all utilities and what components you want.
+#### Use the pre-compiled UX Pattern Library CSS files
+You can use the edX UX Pattern Library as a static CSS base alongside other CSS files. To do so add a reference in the ``<head>`` of your app's HTML
```

Comment:
Solid BETA/heads up note added along with a way to report issues - https://github.com/edx/ux-pattern-library/commit/6b78fe83d696a10ea7bddda980d740c7a82db869.